presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/TestCassandraConnector.java

```diff
@@ -228,7 +231,7 @@ else if (DOUBLE.equals(type)) {
                 else if (REAL.equals(type)) {
                     cursor.getLong(columnIndex);
                 }
-                else if (isVarcharType(type) || VARBINARY.equals(type)) {
+                else if (isVarcharType(type) || VARBINARY.equals(type) || DateType.DATE.equals(type)) {
```

Comment:
Why would i call `cursor.getSlice` for a DATE value (backed by long, in 32-bit int range)?